contents/blog/the-posthog-array-1-42-0.md

```diff
@@ -0,0 +1,126 @@
+---
+date: 2022-11-30
+title: "Array 1.42.0: SUBTITLE HERE"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? Check out [our roadmap](/roadmap) to see what we're working on and what new beta features are available!
+
+## PostHog 1.42.0 release notes
```

Comment:
We need to add upgrade notes here, saying: https://github.com/PostHog/posthog.com/pull/4738


> 
Async migrations 0005-0007 need to be completed before updating to 1.42, if you haven't yet ran them upgrade first to 1.41.4, then run the async migrations, then update to 1.42.

Reply:
nit for both:
> Async migrations 0005-0007 need to be completed before updating to 1.42, if you haven't yet ran them and they aren't available on your current version, then upgrade first to 1.41.4, then run the async migrations, then update to 1.42.